app/views/order_items/_order_item.html.erb

```diff
@@ -5,6 +5,17 @@
         <%= image_tag product.avatar %>
       </div>
       <div class="caption">
+        <h6>
+          <% if product.calories != nil %>
```

Comment:
Use `unless` instead of `if !=` You can even leave out the `nil` and just use `unless product.calories`